soc/arm/nordic_nrf/Kconfig.soc

```diff
@@ -3,4 +3,19 @@
 # Copyright (c) 2016-2018 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+config NFCT_PINS_AS_GPIOS
+       bool "NFCT pins as GPIOs"
+       depends on HAS_HW_NRF_NFCT
+       help
+         Two pins are usually reserved for NFC in SoCs that implement the
+         NFCT peripheral. This option switches them to normal GPIO mode.
+         HW enabling happens once in the device lifetime, during the first
+         system startup. Disabling this option will not switch back these
+         pins to NFCT mode. Doing this requires UICR erase prior to
+         flashing device using the image which has this option disabled.
+
+         NFC pins in nRF52-based DKs: P0.9 and P0.10
+         NFC pins in nRF5340 DK: P0.2 and P0.3
```

Comment:
These pins assignments are properties of SoCs, not DKs, so I would rather use here "nRF52 Series" and "nRF5340" (or "nRF53 Series").
And the pins should be specified as P0.09, P0.02, and P0.03, as that's how they are referred in product specifications.

Reply:
ups, right. For nRF52, I copied the previous names, though. Hope this change won't be an issue to anyone :)

Reply:
@anangl nRF5340 should be better. Possibly also nRF5340 Application core?